### ISSUE TITLE
The |data| argument in vaCreateBuffer() should be const void*

### ISSUE THREAD

**Issue by rosetta-jpn** [intel/libva#597]
This is a minor problem.
The type of |data| argument in vaCreateBuffer() is void*.
If |data| is null, it is not used in the function.
If |data| is not null, it is copied to a created buffer in the function.
In both cases, |data| is not written. That is, like the type of the source in memcpy(), the |data| type should be const void*.
A VA-API application is likely implemented as a variable type for |data| is const void*. 
In fact, chrome has to drop const in implementation around this function due to this type mismatch.

**Comment by XinfengZhang** [comment]
yes, the backend driver implementation of vaCreateBuffer should not change the data though the pointer.
